tripper/triplestore.py

```diff
@@ -294,33 +294,35 @@ def parse(
         self,
         source=None,
         format=None,
-        backend=None,
-        backend_kwargs=None,
+        fallback_backend="rdflib",
+        fallback_backend_kwargs=None,
         **kwargs,  # pylint: disable=redefined-builtin
     ) -> None:
         """Parse source and add the resulting triples to triplestore.
 
         Parameters:
             source: File-like object or file name.
             format: Needed if format can not be inferred from source.
-            backend: If given, use the parse() method of the specified
-                backend instead of the current backend.
-            backend_kwargs: Dict with additional keyword arguments passed
-                when instantiating `backend`.
+            fallback_backend: If the current backend doesn't implement
+                parse, use the `fallback_backend` instead.
+            fallback_backend_kwargs: Dict with additional keyword arguments
+                for initialising `fallback_backend`.
```

Comment:
I do not see where these are tested. According to codecov, they are, so currently it should be OK. But it is important that this is tested on purpose and not "by accident", e.g. by using a backend that right now does not contain parse but might in the future.

Reply:
They are tested in tests/backends/test_collection.py

Reply:
OK, I prefer tests to be a bit more explicit.